Source/Layout/BrickFlowLayout.swift

```diff
@@ -693,9 +663,23 @@ extension BrickFlowLayout {
             }
         }
 
-        isUpdating = true
+        if (insertedIndexPaths.count + deletedIndexPaths.count) > 0 {
+            zIndexer.reset(for: self)
+            for (sectionIndex, section) in sections! {
```

Comment:
It's OK to force unwrap sections. Without them being initialized, there wouldn't be any insert/delete possible
I've added a comment to describe this